api/build.gradle

```diff
@@ -93,6 +95,9 @@ build.mustRunAfter apiDeps
 bootRepackage.mustRunAfter apiDeps
 
 dependencies {
+    // Security-config-starter library
+    implementation project(':starter')
```

Comment:
Let's try another approach:
1. Gradle has some feature, called [source dependency](https://blog.gradle.org/introducing-source-dependencies)
2. At the moment, it doesn't work with private repos, specified by HTTPS URL
But maybe, if we use SSH URL and have SSH access configured properly - it will work

Can you please try it, so we can remove starter package from here

Reply:
implemented in a similar way with github token access